pinot-core/src/main/java/com/linkedin/pinot/core/util/DataTableCustomSerDe.java

```diff
@@ -46,6 +48,26 @@
  */
 public class DataTableCustomSerDe extends DataTableJavaSerDe {
   private static final Logger LOGGER = LoggerFactory.getLogger(DataTableCustomSerDe.class);
+  private BrokerMetrics _brokerMetrics;
+
+  /**
+   * Default constructor for the class.
+   */
+  public DataTableCustomSerDe() {
+    _brokerMetrics = null;
+  }
+
+  /**
+   * Constructor for the class, registers the broker metrics to emit a metric each time
+   * we hit a Java serialization/de-serialization, as the goal is to catch all of them
+   * and replace with custom ser/de.
+   *
+   * @param brokerMetrics Broker metrics
+   */
+  public DataTableCustomSerDe(BrokerMetrics brokerMetrics) {
```

Comment:
DataTable class is in utils already. More-over, this is a temporary way to catch any missing object ser/de and the intention is to this code once we have established that.